### ISSUE TITLE
erp development,

### ISSUE THREAD

**Issue by alvin101** [neilsf/webwind#1]
I have a question.  Do you plan to do more with this system?

I am not a coder  but I would like to work with someone who is, 

Please let me know?

Alvin

**Comment by neilsf** [comment]
Yeah I'm hoping to go on with this project sometime around Spring 2018...  as you can see it is yet in a very early phase and I'm not planning that it will ever become a real ERP, it's just a demo project.
Csaba
